examples/code_review_agent/skills/code-review/scripts/run_staticcheck.sh

```
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -euo pipefail
+
+PACKAGE_PATH="${1:-.}"
+
+mkdir -p out
+
+echo "=== staticcheck ===" > out/staticcheck_results.txt
+
+if command -v staticcheck > /dev/null 2>&1; then
+    staticcheck "$PACKAGE_PATH" 2>&1 >> out/staticcheck_results.txt || true
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)